main.py

```diff
@@ -128,7 +128,8 @@ def check_path(path):
 		       '-net','user',
 		       '-drive','file=%(hdd_path)s,index=0,media=disk%(hdd_lock_arg)s' % locals(),
 		       '-drive','index=1,media=cdrom%(dvd_path_arg)s' % locals(),
-		       '-qmp','tcp:localhost:4444,server,nowait']
+		       '-qmp','tcp:localhost:4444,server,nowait',
+                       '-display','sdl']
```

Comment:
tabs vs. spaces

I think we should switch this repo to spaces overall.
My text editor also can't render tabs and spaces differently.
I also always have it configured for spaces because that's what XQEMU and almost every other project I work on uses.
Anyhow, this is a different discussion.

Reply:
Oh, I did not realize it was using tabs. Mind if my next PR untabifies?

Reply:
Please fixup.

Reply:
Done.